jetty-quickstart/src/main/config/etc/jetty-quickstart.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_10_0.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+
+  <New id="props" class="java.util.HashMap">
```

Comment:
ID space is global so should be called `quickstart.properties` or something more specific and less likely to collide